src/Java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMTE_NuclearReactor.java

```diff
@@ -342,155 +356,121 @@ public boolean turnCasingActive(final boolean status) {
 		return true;
 	}
 
+	public FluidStack[] getStoredFluidsAsArray() {	
```

Comment:
Could have just used `getStoredFluids().toArray(new FluidStack[0])`

Reply:
Fixed in next push.